src/lib/shell/MainLoopSilabs.cpp

```diff
@@ -49,11 +49,19 @@ void ReadLine(char * buffer, size_t max)
             break;
         }
 
+#ifdef BRD4325A
```

Comment:
Why does the read need to be on different sides of WaitForShellActivity depending on the board?  Should probably be documented...

Reply:
Added some comments on the latest commit